followanalytics/fa-issues#5736 Fixed most npm vulnerabilities

firedeathz opened the pull request:
followanalytics/fa-issues#5736

- Updated a ton of packages for each template with npm dependencies
- Updated README

khoussem commented on basic/fa-sdk.js:
```diff
@@ -0,0 +1,156 @@
+FollowAnalytics = {
```
You can add a comment here too, that this file will be overwritten

firedeathz committed:
followanalytics/fa-issues#5736 Fixed most npm vulnerabilities

- Updated a ton of packages for each template with npm dependencies
- Updated README
